spec/workflow/states/wait_spec.rb

```diff
@@ -73,10 +99,23 @@
         end
         expect(state.running?(ctx)).to be_falsey
       end
+
+      it "runs" do
+        workflow.run_nonblock
+        expect(workflow.end?).to eq(false)
+        expect(workflow.context.state_history.size).to eq(0)
+
+        Timecop.travel(Time.now.utc + 100) do
+          workflow.run_nonblock
+        end
+
+        expect(workflow.end?).to eq(true)
+        expect(workflow.context.state_history.size).to eq(2)
+      end
     end
 
-    context "with timestamp" do
```

Comment:
NOTE this context was confusing because it is the same as above, believe it should have been TimestampPath